.flake8

```diff
@@ -1,11 +1,12 @@
 [flake8]
-# Suggested config from pytorch that we can adapat
+# Suggested config from pytorch that we can adapt
 select = B,C,E,F,N,P,T4,W,B9
 max-line-length = 120
 # C408 ignored because we like the dict keyword argument syntax
 # E501 is not flexible enough, we're using B950 instead
+# N812 ignored because import torch.nn.functional as F is PyTorch convention
 ignore =
-    E203,E305,E402,E501,E721,E741,F405,F821,F841,F999,W503,W504,C408,E302,W291,E303,
+    E203,E305,E402,E501,E721,E741,F405,F821,F841,F999,W503,W504,C408,E302,W291,E303,N812,
```

Comment:
Going forward, you can break unrelated changes into a separate commit and use `ghstack` or `git worktree` to open stacked PRs.

Reply:
still need to learn ghstack...